docs/quickstarts/aws.md

```diff
@@ -121,6 +121,13 @@ guide](../configuration-reference/platforms/aws.md).
 
 ### Step 5: Create Lokomotive cluster
 
+Lokoctl uses Terraform behind the scene and it needs SSH access to controller nodes. For SSH to work add at least one SSH private key corresponding to the public keys specified earlier, to your `ssh-agent`.
```

Comment:
```suggestion
Add a private key corresponding to one of the public keys specified in `ssh_pubkeys` to your ssh-agent:
```

- IMO it's irrelevant to mention that lokoctl uses Terraform here. All the user needs to know is that they should add an SSH key to ssh-agent.
- "Lokoctl" should be `lokoctl`.
- "behind the scene" should be "behind the scenes".
- I like using a colon at the end of an instruction that is followed by a code snippet. We are currently inconsistent there but if we should settle on something, I suggest it be a colon rather than a period.